src/components/_templates/Sort/Sort.tsx

```diff
@@ -229,30 +230,28 @@ class Visualiser extends React.Component<Record<string, never>, TVisualiserState
 
   render() {
     return (
-      <div className={VISUALISER.VISUALISER}>
-        <div className={VISUALISER.BUTTONS}>
-          <p className={VISUALISER.TITLE}>{this.state.sorting ? STATUS.SORTING : STATUS.CHOSE_ALGORITHM}</p>
-          <Button classNames={VISUALISER.BUTTON} onClick={this.shuffleItems}>
-            Shuffle
+      <div className={SORT.SORT}>
+        <Menu>
+          <Button href="/" asHref="/" type={BUTTON_TYPE.GREEN}>
+            {"< Back"}
```

Comment:
What's the purpose of curly brackets here?

Reply:
You cannot write a symbol '<' even when using '\' before that

Reply:
Just FYI, you can also use [HTML entities](https://dev.w3.org/html5/html-author/charref) `&lt;`, but this solution is also fine